src/main/scala/com/gu/inspectorlambda/aws/AWSInspector.scala

```diff
@@ -4,10 +4,13 @@ import com.amazonaws.services.inspector.AmazonInspector
 import com.amazonaws.services.inspector.model._
 import com.gu.inspectorlambda.chiefinspector.ChiefInspector.inspectionTagName
 import com.typesafe.scalalogging.StrictLogging
+
 import scala.collection.JavaConverters._
 
 class AWSInspector(val client: AmazonInspector) extends StrictLogging {
 
+  val sleepForConsistencyMillis = 2000
```

Comment:
The name confused me on first read, I see now what this means. If you can think of a clearer name that might be helpful, but not a huge deal.

(Consistency has slightly ambiguous meaning without context and there's no context around this val)

Reply:
Something along the lines of "wait for the delete to take effect" but obviously not a terrible name like that is.